mail/mlmmj/Portfile

```
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           codeberg 1.0
+PortGroup           legacysupport 1.1
+
+codeberg.setup      mlmmj mlmmj RELEASE_1_4_6
+
+name                mlmmj
+version             1.4.6
+categories          mail
+license             MIT
+maintainers         @inucat \
+                    proton.me:inucat
+description         A simple and slim mailing list manager
+long_description    Mlmmj is a simple and slim mailing list manager (MLM) \
+                    inspired by ezmlm. It works with many different Mail \
+                    Transport Agents (MTAs) and is simple for a system \
+                    adminstrator to install, configure and integrate with \
+                    other software. As it uses very few resources, and \
+                    requires no daemons, it is ideal for installation on \
+                    systems where resources are limited, such as Virtual \
+                    Private Servers (VPSes).
+
+checksums           rmd160  1f462719550aa4cb2410718d30979ebb99c2bb4f \
+                    sha256  5cd877703c80ab07f9ae845b89a34c032c31d290f4a311f6c8b9fe52ca508822 \
+                    size    281169
+patchfiles          include-xlocale_h.diff \
+                    remove-disabled-test.diff
+use_autoreconf      yes
+configure.args       --disable-tests
```

> ```
> CC       checkwait_smtpreply.o
> ```

These types of "silent rules" hide the actual compiler invocation that was used, making it impossible to verify by looking at a build log whether the right compiler and flags were used. Please disable them so that we can see the full commands:

```suggestion
configure.args       --disable-silent-rules \
                     --disable-tests
```

---

By the way why this is not set by default for makefiles builds by the base or PG?